google/resumable_media/_helpers.py

```diff
@@ -349,6 +362,16 @@ def _get_checksum_object(checksum_type):
         raise ValueError("checksum must be ``'md5'``, ``'crc32c'`` or ``None``")
 
 
+def _get_connection_error_class():
+    """Get the exception error class.
+
+    This is a separate function for testing purposes."""
```

Comment:
Clarify "testing purposes" with detecting if requests is being used.

Reply:
+1, good to explain this layer of indirection.